spec/integration/snapshot_query_examples_spec.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe 'Snapshot Query Examples' do
+  require_topology :replica_set, :sharded
+  require_stress
```

Comment:
I don't think these tests will be executed when you `require_stress` but the file is not under `spec/stress`:

```
.evergreen/run-tests.sh:  bundle exec rspec spec/integration/fork*spec.rb spec/stress/fork*spec.rb
.evergreen/run-tests.sh:  bundle exec rspec spec/integration/fork*spec.rb spec/stress
```

Either this invocation needs to be adjusted or the file needs to be moved under spec/stress.

Reply:
done